lms/static/js/student_account/views/LoginView.js

```diff
@@ -93,16 +86,26 @@
                 },
 
                 resetEmail: function() {
+                    var email = $('#password-reset-email').val(),
+                        successMessage;
                     this.element.hide(this.$errors);
                     this.resetMessage = this.$resetSuccess.find('.message-copy');
+
+                    successMessage = HtmlUtils.interpolateHtml(
+                        gettext('{paragraphStart}You entered {email}. If this email address is associated with an edX account, we will send a message with password reset instructions to this email address.{paragraphEnd}' +
+                        '{paragraphStart}If you do not receive a password reset message, verify that you entered the correct email address, or check your spam folder. ' +
+                        '{paragraphStart}If you need further assistance, {anchorStart}contact technical support{anchorEnd}.'), {
+                            paragraphStart: HtmlUtils.HTML('<p>'),
+                            paragraphEnd: HtmlUtils.HTML('</p>'),
+                            email: email,
+                            anchorStart: HtmlUtils.HTML('<a href="' + this.supportURL + '">'),
```

Comment:
@tasawernawaz on your sandbox this link is pointing to https://tasawernawaz.sandbox.edx.org/[object%20Object]